pyproject.toml

```diff
@@ -88,7 +88,7 @@ dev = [
 ]
 docs = [
     "canonical-sphinx[full]>=0.4.0",
-    "docutils<0.20", # Frozen until we can update sphinx dependencies.
+    "docutils<0.22", # Frozen until we can update sphinx dependencies.
```

Comment:
Pretty sure this is no longer true since `docs/.sphinx/pinned-requirements.txt` also got bumped.  Maybe @lengau can confirm?

Reply:
This had also been a standard dep when this comment was added: https://github.com/canonical/snapcraft/commit/d4078a013856a9161b5da94bad1df1c9d823bf5a

Reply:
Yeah we can nuke this and make it `>=0.21` instead.